packages/text/src/kits/default-kit.ts

```diff
@@ -54,11 +54,7 @@ export const DefaultKit = Extension.create<DefaultKitOptions>({
       Highlight.configure({
         multicolor: false
       }),
-      Typography.configure({}),
-      Link.configure({
```

Comment:
You cannot just remove this extension:
1. Link button in toolbar will be broken
2. Existing content that contains links will be broken